server/src/test/kotlin/fi/oph/ludos/assignment/AssignmentControllerTest.kt

```diff
@@ -551,4 +551,82 @@ class AssignmentControllerTest(@Autowired val mockMvc: MockMvc) {
         mockMvc.perform(postAssignment(testAssignmentStr)).andExpect(status().isUnauthorized())
         mockMvc.perform(updateAssignment(1, testAssignmentStr)).andExpect(status().isUnauthorized())
     }
+
+    private fun markAndQueryFavorites(localAssignmentIdToFavoriteAsOpettaja: Int) {
```

Comment:
Aloittaisin tän nimen `test`, ni sit on vilkaisullakin selvää, että se testaa asioita

```suggestion
    private fun testMarkingAndQueryingFavorites(localAssignmentIdToFavoriteAsOpettaja: Int) {
```